content/purgatorio/launch-and-mantain-the-system/automation-and-reproducibility.md

```diff
@@ -1,5 +1,242 @@
+
+Here you find another interesting comparison of the different platforms:
+
+- [AutoML Software Comparison](https://aimultiple.com/automl-software)
+
+--- 
+
+If you want to get started with the Google's AutoML platform, here you can find a good series of tutorials about it: 
+
+- [AutoML Vision Beginner's guide](https://cloud.google.com/vision/automl/docs/beginners-guide)
+- [AutoML Tables Beginner's guide](https://cloud.google.com/automl-tables/docs/beginners-guide)
+- [AutoML Natural Language Beginner's guide](https://cloud.google.com/natural-language/automl/docs/beginners-guide)
+- [AutoML Translation beginner's guide](https://cloud.google.com/translate/automl/docs/beginners-guide)
+
+Just try it out!
+
+::: warning
+Often AutoMl jobs consume a lot of computational resources (since it often requires a search in the model architectures space), so watch your back with the billing of the platform that you choose!
+:::
+
+## No Free Lunch and Black Boxes
+
+It may seem too good to be true, right? There are actually some aspects to be careful about when choosing an AutoML approach. 
+
+- Not all data types are suitable / supported by the currently available frameworks (autoML works best with tabular data). More complex data formats (like images, videos, audio...) aren't often considered in modern AutoML frameworks.
+- However, it is important to know the data well, to know what information (often called signals) it contains, and to know what information we want to derive from it. 
+- There is no such thing (as usual) as a _silver bullet_, so don't expect spectacular results in a magical way, on every problem you approach with AutoML. 
+- It's usually computationally intensive, this can rapidly inflate the billing costs of the platform you're working on.  
+
+Another **issue** you must consider when applying AutoML techniques, is the **opacity** that you bring to the process.
+
+The concept of "[**Black Box**](https://towardsdatascience.com/the-black-box-metaphor-in-machine-learning-4e57a3a1d2b0)", linked to more complex ML algorithms (such as neural networks) makes the resulting ML models "opaque" and therefore difficult to explain, **should be equally considered for all steps of the Data Science process to which some automatic decision applies**.
+
+For example:
+
+- If some data preprocessing steps are decided from the machine, _why_ are they like that?
+- Is it possible that these choices will affect the models that will then be trained on the data? If yes, _how_?
+- If some features have been dropped and others have been chosen for the training, _why_? 
+- If a certain model was chosen to be trained on data, _why_ that model? And if the data changed over time, would that model be able to _adapt_ to the new situation (after appropriate re-training)?
+
+As you can see, as there are many pros in using AutoML, there are also various issues that can be problematic.
+
+::: tip
+**What does Virgiloo think of AutoML?**
+
```

Comment:
guiding him --> try to make sentences like this gender neural I would say "guiding them"!